Orange/widgets/visualize/owsieve.py

```diff
@@ -192,6 +187,7 @@ def set_data(self, data):
         self.vizrank_button.setEnabled(
             self.data is not None and len(self.data) > 1 and
             len(self.data.domain.attributes) > 1)
+        self.vizrank_button.setVisible(self.data is not None and not self.data.is_sparse())
```

Comment:
I prefer `setEnabled` than `setVisible`. Currently, when no data is present, we disable it not hide it. Let's do the same when the data is sparse.